build.sc

```diff
@@ -5,11 +5,12 @@ import publish._
 
 import $ivy.`com.lihaoyi::mill-contrib-bloop:$MILL_VERSION`
 
-val scala212 = "2.12.10"
+val scala212 = "2.12.8"
```

Comment:
A Scala downgrade? 😳

Reply:
Yes, just for local technical reasons, I shouldn't commit this, thanks.